src/ucp/api/ucp.h

```diff
@@ -1189,6 +1227,31 @@ unsigned ucp_worker_progress(ucp_worker_h worker);
 
 
 /**
+ * @ingroup UCP_WORKER
+ * @brief Poll for endpoints that are ready to consume streaming data.
+ *
+ * This non-blocking routine returns endpoints on a worker which are ready
+ * to consume streaming data. The ready endpoints are placed in @poll_eps array,
+ * and the function return value indicates how many are there.
+ *
+ * @param [in]   worker    Worker to poll.
+ * @param [out]  poll_eps  Pointer to array of endpoints, should be
+ *                         allocated by user.
+ * @param [in]   max_eps   Maximal number of endpoints which should be filled
+ *                         in @a poll_eps.
+ * @param [in]   flags     Reserved for future use.
+ *
+ * @return Negative value indicates an error according to @ref ucp_status_t.
+ *         On success, non-negative value (less or equal @a max_eps) indicates
+ *         actual number of endpoints filled in @a poll_eps array.
+ *
+ */
+ssize_t ucp_stream_worker_poll(ucp_worker_h worker,
```

Comment:
1. how creates the 'recv' EP?
2. do you have API to create these? or they are delivered from this API (created in UCX inside)
3. when sending, how do I send to a specific EP address?

Reply:
@rosenbaumalex there are 2 ways:
1) use existing ep_create API + out of band address exchange (like MPIs do)
2) use new connect/accept API https://github.com/openucx/ucx/pull/1720

Reply:
@rosenbaumalex - do you think we have gaps in our documentation ? If so, we would love to address those.